tetris.py

```diff
@@ -221,6 +225,9 @@ def display_highscores():
 def new_game():
     if tetris_control.game is not None:
         tetris_control.game.status = GameStatus.game_over
+    if treeview:
```

Comment:
I don't think this works. Do you want to say "if the treeview is packed"? If you do, make a function that returns `True` when a widget is packed and `False` when it isn't, using the code you already wrote for this.

Reply:
The code works!